workflows/postprocessing.nf

```diff
@@ -120,7 +120,7 @@ process genotypeGVCF {
     script:
     def familyId = meta.familyId
     def args = task.ext.args ?: ''
-    def argsjava = task.ext.args ?: ''
```

Comment:
Change needed in Import process too,   right ?

Reply:
Correct, the problem appears for many processes actually, so I will use another PR for the other ones as this one is starting to be a bit fat. 
I created a github issue so that we don't forget:
https://github.com/Ferlab-Ste-Justine/Post-processing-Pipeline/issues/14